<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DialogShowFile</class>
    <widget class="QDialog" name="DialogShowFile">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <widget class="QDialogButtonBox" name="buttonBox">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>260</y>
                    <width>341</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="orientation">
                <enum>Qt::Horizontal</enum>
            </property>
            <property name="standardButtons">
                <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
        </widget>
        <widget class="QWidget" name="">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>30</y>
                    <width>341</width>
                    <height>221</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="minimumSize">
                                    <size>
                                        <width>51</width>
                                        <height>34</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>Name:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label_name">
                                <property name="text">
                                    <string>TextLabel</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                            <widget class="QLabel" name="label_3">
                                <property name="text">
                                    <string>Author:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label_author">
                                <property name="text">
                                    <string>TextLabel</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                        <item>
                            <widget class="QLabel" name="label_5">
                                <property name="text">
                                    <string>Date:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label_date">
                                <property name="text">
                                    <string>TextLabel</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                        <item>
                            <widget class="QLabel" name="label_7">
                                <property name="text">
                                    <string>Size:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label_size">
                                <property name="text">
                                    <string>TextLabel</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                        <item>
                            <widget class="QLabel" name="label_9">
                                <property name="text">
                                    <string>Data:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QTextEdit" name="textEdit_data"/>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>DialogShowFile</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>220</x>
                    <y>275</y>
                </hint>
                <hint type="destinationlabel">
                    <x>199</x>
                    <y>149</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>DialogShowFile</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>220</x>
                    <y>275</y>
                </hint>
                <hint type="destinationlabel">
                    <x>199</x>
                    <y>149</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
